source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,748 @@
+
+.. command-block::
+   :no-exec:
+
+   qiime demux summarize --help
+
+Based on the documentation, we should pass the demultiplexed sequences that we imported as the ``--i-data`` argument, since this takes a ``SequencesWithQuality]`` semantic type, and that’s the type of data we imported. We’ll specify the location we want the visualization by passing the output path to ``--o-visualization``. However, to speed up the command process, we’ll change the ``--p-n`` parameter to 1000. This means that rather than resampling the sequences 10000 times (the default number) to get the quality score, we’ll only re-sample them 1000 times.
+
+The help documentation is a good reference for any command, and the first place to look if you’re getting errors, especially errors about parameters.
+
+.. command-block::
+
+   qiime demux summarize \
+     --i-data ./demux_seqs.qza \
+     --p-n 1000 \
+     --o-visualization ./demux_seqs.qzv
+
+You can view the .qzv visualization file at `view.qiime2.org`_. Just drag and drop the file into the viewer window.
+
+.. question::
+
+   1. After demultiplexing, which sample has the lowest sequencing depth?
+   2. What is the median sequence length?
+   3. What is the median quality score at position 125?
+
+
+Sequence quality control and feature table
+==========================================
+
+There are several ways to construct a feature table in QIIME 2. The first major separation is between Operational Taxonomic Units (OTUs) and Absolute Sequence Variants (ASVs). OTUs have been widely used in microbiome research since the mid 2010s, and assign sequences to taxonomic clusters either based on a reference database or de novo assignment. QIIME 2 offers clustering through :doc:`q2-vsearch<otu-clustering>` and `q2-dbOTU_` plug-ins, currently.
+
+ASVs are a more recent development and provide better resolution in features than traditional OTU-based methods. ASVs can separate features based on differences of a single nucleotide in sequences of 400 bp or more, a resolution not possibly even with 99% identity OTU clustering. QIIME 2 currently offers denoising via `Dada2`_ (``q2-dada2``) and `Deblur`_ (``q2-deblur``). The major differences in the algorithms and motivation for denoising are nicely described in `Nearing et al, 2018`_.
+
+It is worth noting in either case that denoising to ASVs and clustering to OTUs are seperate, but parallel steps. A choice should be made for a single pathway: either denoising or OTU based clustering; it is not recommended to combine the steps.
+
+In this tutorial, we’ll denoise using Deblur on single ended sequences. those interested in Dada2 may find the :doc:`moving pictures tutorial  <moving-pictures/>` and :doc:`Atacama soil tutorial <atacama-soils>`. An example of using Deblur with paired end reads can be found in the :doc:`Alternative methods of read joining <read-joining/>` tutorial.
+
+Quality filtering
```

Comment:
I am certainly not opposed to deblur here, but, one thing to keep in mind in a workshop environment is that the q2-deblur pipeline has a few more steps than the q2-dada2 pipeline. Food for thought. Also, we don't necessarily need to choose one vs the other in the doc itself, we can do something similar to the MP tutorial where we demonstrate _both_ pipelines in the doc, but only execute one pipeline in an instructional scenario.

Reply:
Semi person preference, semi what I want my students to use when I make them go through this tutorial because I prefer Deblur to Dada2, but it doesn't really matter to me here.

@nbokulich thoughts?

Reply:
deblur is faster, which is a bonus for workshops (though I don't know how that stacks up with the extra QC step)

Otherwise I agree with @thermokarst the fewer steps the better for a workshop...

Reply:
It looks like in this case, Dada2 is quicker than Deblur. So, I'm going to update the tutorial to use Dada2 for the workshop. Might take me a day or two and will slightly shift questions but shouldnt change conclusions.